app/api/logging/__init__.py

```diff
@@ -1,4 +1,4 @@
-import datetime
+import json
```

Comment:
For tests of the various utils I've ported over, perhaps a follow-up PR to cover that might be better? Mostly because I've got more changes ontop of this PRs in subsequent ones to the logging